app/models/review.rb

```diff
@@ -29,6 +29,13 @@ def get_score
     total
   end
 
+  def self.order_by_vote
+    select('reviews.*, sum(votes.value) as score')
+    .joins(:votes)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.